lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -99,7 +100,7 @@
     - A list of disks to add.
     - 'Valid attributes are:'
     - ' - C(size_[tb,gb,mb,kb]) (integer): Disk storage size in specified unit.'
-    - ' - C(type) (string): Valid value is C(thin) (default: None).'
+    - ' - C(type) (string): Valid value is C(thin) or C(eagerzeroedthick) (default: None).'
```

Comment:
In the text? or is there a way to add a yaml field to the doc to indicate this? (Trying to search for the answer now)